src/scss/_action.scss

```diff
@@ -34,58 +34,44 @@
     justify-content: center;
 
     //content unique to this instance of c-action__link
-    background-color: $orange;
+    background-color: darken($green-brand, 2%);
```

Comment:
One spot that isn't sitting just right for me.

![screen shot 2016-09-01 at 9 40 24 am](https://cloud.githubusercontent.com/assets/8847861/18169491/4ef5bc60-7028-11e6-998a-ab38f36969b4.png)

I wonder about the line height on these paragraphs. The headings are pretty tight on the copy but there's a lot of air in line height of the text. I might play with those distances a little bit.

Other than that, I really can't criticize the design. It looks great. Much better than I could do. 😄